.idea/misc.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="ASMIdeaPluginConfiguration">
+    <asm skipDebug="false" skipFrames="false" skipCode="false" expandFrames="false" />
+    <groovy codeStyle="LEGACY" />
+  </component>
```

Comment:
Unrelated change

Reply:
Oh, sorry.

Reply:
It's so cumbersome to have IDEA settings saved automatically.